edi_pdf2data_oca/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Pdf2data Import",
+    "summary": """
+        Module that allows to import data from a pdf""",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "CreuBlanca,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/edi",
+    "depends": ["edi_oca"],
+    "external_dependencies": {
+        "python": ["dateparser", "unidecode"],
```

Comment:
Why are you depending on unidecode? I don't see it used here. FYI, We are considering removing it from invoice2data because of license incompatability.

Reply:
True! Removed